cloud/lxc/lxc_container.py

```diff
@@ -57,7 +57,6 @@
         description:
           - Path to the LXC configuration file.
         required: false
-        default: /etc/lxc/default.conf
```

Comment:
instead of removing the default, change its value to null or none, e.g. :`default: null`